internal/controllers/nodefeaturediscovery_reconciler.go

```diff
@@ -330,23 +330,25 @@ func (nfdh *nodeFeatureDiscoveryHelper) handlePrune(ctx context.Context, nfdInst
 		return false, fmt.Errorf("failed to get nfd-prune job: %w", err)
 	}
 
-	deleteJob := false
+	//deleteJob := false
 	var returnErr error
 	done := false
 	if pruneJob.Status.Succeeded > 0 {
-		deleteJob = true
+		//deleteJob = true
 		done = true
 	}
 	if pruneJob.Status.Failed > 0 {
-		deleteJob = true
+		//deleteJob = true
 		returnErr = fmt.Errorf("prune job's pod has failed")
 	}
-	if deleteJob {
-		err = nfdh.jobAPI.DeleteJob(ctx, pruneJob)
-		if err != nil {
-			return false, fmt.Errorf("failed to delete nfd-prune job: %w", err)
+	/*
```

Comment:
Why leave this line commented and not simply remove it?

Reply:
my bad, forgot to remove it, fixed